client/homebrew/pages/basePages/listPage/brewItem/brewItem.less

```diff
@@ -59,6 +59,44 @@
 		white-space: nowrap;
 		display: inline-block;
 		font-weight: bold;
```

Comment:
Let's add `borderColor:currentColor` here, and delete border-color declarations along the way, since they all use the same color.

Reply:
Applied in the latest commit.